src/libsyntax/parse/lexer/tests.rs

```diff
@@ -38,7 +39,7 @@ fn t1() {
         let mut string_reader = setup(
             &sm,
             &sh,
-            "/* my source file */ fn main() { println!(\"zebra\"); }\n".to_string(),
+            "/* my source file */ fn main() { println!(\"zebra\"); }\n".to_owned(),
```

Comment:
Prefer `.to_string()` -- it says what the resulting type is, better for readability.

Reply:
Fine, okay.